### DISCUSSION TITLE
Application error: a client-side exception has occurred (see the browser console for more information).

### DISCUSSION THREAD

**Discussion by MaxCode012** [vercel/community#7366] [Bug Report] [answered]
### Page to Investigate

https://thread-sphere-ten.vercel.app/

### Steps to Reproduce

Im getting this error and i cant escape it i even wrote git checkout to the oldest version and this still popped up. Im sure this is a bug. 4days ago when i last coded everything was fine. In production everything is fine aswell. Maybe postgres and prisma are the problem but im sure i did everything according to the docs here is my repo please help https://github.com/MaxCode012/ThreadSphere ignore my 8th commit because it didnt solve anything
![Captură ecran (79)](https://github.com/vercel/community/assets/130999640/9ae5160c-c903-4b86-ac61-ddf8b222ecc4)

**Comment by pawlean** [comment, marked as answer]
Hi @MaxCode012!

From the screenshot, it appears the error is related to hydration (worth digging deep on this for your learning!), which occurs when React attempts to attach event listeners to the existing server-rendered HTML during the initial client render.

It's worth checking:

- Ensure all dependencies, especially React, React-DOM, and Next.js, are up to date and compatible.
- Check `package.json` and run `npm update` or `yarn upgrade`.
- Ensure server-side data fetching methods (like getServerSideProps or getStaticProps) return the expected data.
- Check for discrepancies between server-rendered and client-rendered HTML

**Reply by MaxCode012** [reply]
i did everything it still doesnt work. Everytime i create a nextjs project it shows me this and it is completely unusable. First second it show me the content of the page then shows this text.

[a comment by an automated account is not shown]
